src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/MflStatSpec.groovy

```diff
@@ -353,4 +370,35 @@ class MflStatSpec extends HealthCheckSpecification {
             flowHelperV2.deleteFlow(flow.flowId)
         }
     }
+
+    def switchIsConnectedToFl(SwitchId switchId, Boolean mgmt, Boolean stats) {
+        Wrappers.wait(WAIT_OFFSET / 2) {
+            def switchConnectionInfo = northboundV2.getSwitchConnections(switchId)
+            if (mgmt && stats) {
+                assert !findMgmtFl(switchConnectionInfo).empty
+                assert !findStatFl(switchConnectionInfo).empty
+            } else if (mgmt && !stats) {
+                assert !findMgmtFl(switchConnectionInfo).empty
+                assert findStatFl(switchConnectionInfo).empty
+            } else if (!mgmt && stats) {
+                assert findMgmtFl(switchConnectionInfo).empty
+                assert !findStatFl(switchConnectionInfo).empty
+            } else if (!mgmt && !stats) {
+                assert findMgmtFl(switchConnectionInfo).empty
+                assert findStatFl(switchConnectionInfo).empty
+            }
+        }
+    }
+
+    def findStatFl(SwitchConnectionsResponse switchConnections) {
+        return switchConnections.connections.findAll {
+            it.regionName.contains('stats') || it.connectMode == SwitchConnectMode.READ_ONLY.toString()
```

Comment:
`it.regionName.contains('stats')`
I don't like this. Why is this required in addition to `it.connectMode == SwitchConnectMode.READ_ONLY.toString()` ? If this condition for some reason is not enough, then look into 
`flHelper.fls.findAll { it.mode == FloodlightConnectMode.RO }*.region` to get stats regions.
Relying on magic strings is a bad practise

Reply:
Actually, it is more than enough.
`it.regionName.contains('stats')` - deleted.